app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -149,6 +149,7 @@ class ScreenController(private val sessionRepo: SessionRepo) {
             MenuInteractionMonitor.menuClosed()
         }
 
+        TelemetryIntegration.INSTANCE.userShowsHidesDrawerEvent(toShow)
```

Comment:
The original intent of this telemetry is, “User intentionally opens overlay” (i.e. presses menu or back button) or “User intentionally closes overlay without an action” – so I think this PR doesn’t capture that because it's capturing all open and closes.